arangodb-http/src/it/scala/io/funkode/arangodb/http/ArangoJsonIT.scala

```diff
@@ -67,9 +67,20 @@ trait ArangoExamples:
   val secondCountries = List(Country("🇦🇫", "Afghanistan"), Country("🇦🇬", "Antigua and Barbuda"))
 
   val politics = GraphName("politics")
+  val geography = GraphName("geography")
   val allies = CollectionName("allies")
+  val geographyCollection = CollectionName("geography")
+  val geographyCollection2 = CollectionName("geography2")
```

Comment:
I think this is not used on this file, maybe you wanted to use on line 81?

Reply:
Nice catch. Updated the code